sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/json/SensitivityJson.java

```diff
@@ -1,50 +0,0 @@
-/**
- * Copyright (c) 2021, RTE (http://www.rte-france.com)
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/.
- */
-package com.powsybl.sensitivity.json;
-
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.google.common.base.Supplier;
-import com.google.common.base.Suppliers;
-import com.powsybl.commons.extensions.Extension;
-import com.powsybl.commons.extensions.ExtensionJsonSerializer;
-import com.powsybl.commons.extensions.ExtensionProviders;
-import com.powsybl.commons.json.JsonUtil;
-import com.powsybl.sensitivity.SensitivityAnalysisParameters;
-
-/**
- * @author Sebastien Murgey <sebastien.murgey at rte-france.com>
- */
-public final class SensitivityJson {
```

Comment:
Note for later: We need to list the renaming as a breaking change, since it's public, although probably not much used outside.